Gruntfile.js

```diff
@@ -167,8 +167,9 @@ module.exports = function (grunt) {
             },
         },
         exec: {
-            'webpack-dev': `"${webpackPath}" --config-name dev`,
-            'webpack-dev-mv3': `"${webpackPath}" --config-name dev-mv3`,
+            'esbuild-dev': `node esbuild.js`,
+            'esbuild-dev-mv3': `node esbuild.js --env devM3`,
+            'esbuild-prod': `node esbuild.js --env prod`,
             'webpack-prod': `"${webpackPath}" --config-name prod`,
```

Comment:
Did you intend to leave `webpack-prod` here?

Reply:
Yes; we're using it in the report build. I did verify that report package was working but didn't want to include that in this PR so decided to leave that unchanged (and hence leaving the webpack-prod command here).